test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 # This is an end to end test of multi cluster deployement.  
-# For readable instructions, please visit https://github.com/kubestellar/kubestellar/tree/ks-0.20/docs/content/v0.20
+# For readable instructions, please visit ../../../docs/content/v0.20
```

Comment:
So what exactly is the definition and role of the  README under the test directory ?

Reply:
Most importantly: not changed in this PR.
FYI, it is about how to run the tests.

Reply:
I accept the "not changed"  But I prefer to point to the test README as the source of truth (will ad @pdettori link as an additional optional source of info).